web/js/containers/tour.js

```diff
@@ -516,12 +527,47 @@ const mapDispatchToProps = (dispatch) => ({
           type: BULK_PALETTE_RENDERING_SUCCESS,
           rendered: obj.rendered,
         });
+        dispatch({
+          type: BULK_PALETTE_PRELOADING_SUCCESS,
+          tourStoryPalettes: obj.rendered,
+        });
         dispatch({ type: LOCATION_POP_ACTION, payload: location });
       });
     } else {
       dispatch({ type: LOCATION_POP_ACTION, payload: location });
     }
   },
+  preProcessStepLink: async (search, config, promiseImageryForTour) => {
+    search = search.split('/?').pop();
+    const parameters = util.fromQueryString(search);
+    let layersA = [];
+    let layersB = [];
+    const promisesParams = [];
+
+    if (parameters.l) {
+      layersA = layersParse12(parameters.l, config);
+      layersA = uniqBy(layersA, 'id');
+      layersA = layersA.filter((layer) => !layer.custom && !layer.disabled);
+      promisesParams.push({ layersA, dateString: parameters.t });
+      if (parameters.l1) {
```

Comment:
Is there a reason the parameters.l1 IF is inside the parameters.l IF? Why not 2 separate if statements?

Reply:
I don't think there can be an instance where there can be `l1` without `l`, so I nested them to save on an unnecessary comparison. I'm happy to bring it out so it stays unnested though